pixi/rollup.config.js

```diff
@@ -0,0 +1,37 @@
+import replace from '@rollup/plugin-replace';
+import resolve from '@rollup/plugin-node-resolve';
+import commonjs from '@rollup/plugin-commonjs';
+
+const config = require('./config.js');
```

Comment:
Nit: consistently use `require` or `import`.

Reply:
This is intentional to get rollup to work (if I remember correctly).